### PR TITLE
ncompress 4.2.4.4 (new formula)

### PR DESCRIPTION
**Pull request by sjackman**
Fast, simple LZW file compressor

### CODE REVIEW

**Comment by sjackman** [comment]
GitHub not notable enough, otherwise it passed CI.

```
Error: 1 problem in 1 formula
homebrew/dupes/ncompress:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```
https://bot.brew.sh/job/Homebrew%20Dupes%20Pull%20Requests/363/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_homebrew_dupes_ncompress___new_formula/

File: ncompress.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Ncompress < Formula
+  desc "Fast, simple LZW file compressor"
+  homepage "https://github.com/vapier/ncompress"
+  url "https://github.com/vapier/ncompress/archive/v4.2.4.4.tar.gz"
+  sha256 "2670439935e7639c3a767087da99810e45bc3997d0638b3094396043571e5aec"
+  head "https://github.com/vapier/ncompress"
+
+  def install
+    system "make"
+    bin.install "compress"
```
Comment (MikeMcQuaid) [inline ncompress.rb]: Any `make install` we can use here? If not, would be 🆒 to request one.

File: ncompress.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    man1.install "compress.1"
+    prefix.install_metafiles
```
Comment (MikeMcQuaid) [inline ncompress.rb]: Think this should get done automatically given we don't `cd` elsewhere.

File: ncompress.rb (hunk 1 above)
Comment (sjackman) [inline ncompress.rb]: Done.

File: ncompress.rb (hunk 2 above)
Comment (sjackman) [inline ncompress.rb]: Ah, I didn't realize that `prefix.install_metafiles` is run automatically. Nice.

**Comment by sjackman** [comment]
Thanks for your feedback, Mike.

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
Thanks @sjackman for the pull request!

**Comment by sjackman** [comment]
Thanks for merging!

**Pull request closed without merging**